scripts/hail_batch/king_tob_samples/king_tob.py

```diff
@@ -0,0 +1,44 @@
+"""
+Estimate kinship coefficient using KING on TOB samples from the HGDP/1KG dataset.
+"""
+
+import hail as hl
+import pandas as pd
+from analysis_runner import bucket_path, output_path
+
+HGDP1KG_TOBWGS = bucket_path(
+    '1kg_hgdp_densified_pca_new_variants/v0/hgdp1kg_tobwgs_joined_all_samples.mt'
+)
+
+
+def query():
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(HGDP1KG_TOBWGS)
+    mt = mt.filter_cols(mt.s.contains('TOB'))
```

Comment:
Why do you only want to keep TOB samples? While it might represent an interesting test, it would be really surprising to see any samples being flagged as related. Wouldn't it be more relevant to keep the 1KG samples and see whether the trios are correctly identified as such?

Reply:
Why would it be surprising to see samples flagged as being related? We did see some samples being flagged as related using pc_relate (see https://main-web.populationgenomics.org.au/tob-wgs/tob_wgs_hgdp_1kg_nfe_pca_new_variants/v6/filtered_samples_in_gnomad.html). 

The reason I kept TOB samples only is because I already ran KING on them in the publicly available HGDP + 1kG dataset (as done here: https://github.com/populationgenomics/ancestry/blob/52cf3021503a0458e8a0a0c0f0125db834f4cad6/scripts/hail_batch/pc_relate_nfe_samples/pc_relate_nfe.py). 

One thing that might be useful is seeing how randomly-chosen variants compares to ld-pruned variants, so I'd be interested to run this again on NFE samples checking that.

Reply:
> Why would it be surprising to see samples flagged as being related?

It would be surprising because related individuals were excluded in the TOB-WGS study.

> The reason I kept TOB samples only is because I already ran KING on them in the publicly available HGDP + 1kG dataset

Did KING correctly identify all the trios present there?

Reply:
Interesting, so was a relatedness metric already run on TOB samples? From a slack message conversation with Daniel, he asked if there were any traces of cryptic relatedness between TOB samples, so I was under the impression that wasn't run already.

When run on the NFE subset, KING fails to identify 7 samples that are from known trios. However, the two individuals from trios (HG01629 and HG01696) that weren't coming up using `pc_relate` do show up as being related in this test.

Reply:
> I was under the impression that wasn't run already.

Oh, I don't think it has been run yet (and I'm all for running it), but I'd still find it surprising if you actually found a related sample. I just thought it would be good to also include 1KG to see that we do find the actually related samples (as otherwise it's hard to be sure that things are working as expected when there are no "hits").

If you're confident with the code (if it's similar enough to the one you ran on 1KG), it's of course fine to just run this on the TOB-WGS cohort.

> KING fails to identify 7 samples that are from known trios.

That is so strange! We should definitely dig further into that.

Reply:
Yeah, it is strange. Reading up on the KING method, it seems like you don't need ld-pruned variants, but I wonder if this actually changes things. I think it would be interesting to see how the NFE samples compare in this run with those in the randomly-sampled run.

Reply:
I added in the NFE samples in addition to the TOB samples now